qhub_ops/template/{{ cookiecutter.repo_directory }}/infrastructure/kubernetes.tf

```diff
@@ -113,6 +113,7 @@ module "qhub" {
 
   name      = "qhub"
   namespace = var.environment
+  kubessh-namespace = var.kubessh-environment
```

Comment:
Does this needs to be different from the default namespace?

Reply:
Let me check the docs.

Reply:
Also, just to be clear by default I mean the one we use to deploy everything:  `dev` namespace.